### PR TITLE
Admin Page: add the new Apps and Plans landing pages

### LINKED ISSUE #4303: Admin: Apps landing page
The Apps landing page within Jetpack React!

image

(layered files available in .sketch format - just ask me for them if you need them!)

Perhaps this week it would be wise for me to export all necessary jpgs, pngs, and svgs, so you devs don't have to dig around within sketch and slow your workflow down. 

note: content in these screenshots are not finalized at this time.

cc @dereksmart @eliorivero

### LINKED ISSUE #4304: Admin: Plans landing page(s)
The landing pages to be utilized within Jetpack React:

**1) FREE: Jetpack user on free "plan"**

image

**2) PREMIUM: Jetpack user on premium plan**

image

**3) PRO: Jetpack user on pro plan**

image

Will try and get all the pngs, jpgs, and svgs primed and ready before the end of the week. 

_note: content in these screenshots are not finalized at this time._ 

cc @dereksmart @eliorivero

### PR DESCRIPTION
**Pull request by eliorivero**
Fixes #4303.

#### Changes proposed in this Pull Request:
- add the new Apps landing page

### CODE REVIEW

File: _inc/client/apps/index.jsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+					</h2>
+
+					<p className="jp-jetpack-landing__description">
+						{ __( 'Some secondary headline here for consistency.' ) }
+					</p>
+
+					<div className="jp-jetpack-landing__img-text">
+						<div className="jp-jetpack-landing__column">
+							<h2>{ __( 'Feel the performance' ) }</h2>
+							<p>{ __( "All the WordPress apps are built for spped. You'll notice the difference in performance immediately, with near-instant page-loads and less waiting around." ) }</p>
```
Comment (zinigor) [inline _inc/client/apps/index.jsx]: Typo in the word `speed`.

File: _inc/client/apps/index.jsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+							{ __( 'Browse WordPress.com' ) }
+						</Button>
+					</div>
+					<div className="jp-jetpack-landing__column">
+						<img src={ imagePath + '/apps/community2x' } />
+					</div>
+				</div>
+
+				<div className="jp-jetpack-landing__footer">
+					<h2 className="jp-jetpack-landing__title" title="Inspiration strikes any time, anywhere.">
```
Comment (zinigor) [inline _inc/client/apps/index.jsx]: Should all `h2` elements have a title? If so, let's make them all translated as well. And if not, maybe we can remove it?

**Comment by zinigor** [comment]
Code-wise looks great except for several nitpicks, well done, thank you!
And one more thing - images do not load for me without the extension, I have tried both Firefox and Chrome. When I edit the source and add `.png` at the end, they appear as they should.

**Comment by eliorivero** [comment]
Solved issues pointed by @zinigor.

**Pull request merged**